cpp/src/transform/transform.cpp

```diff
@@ -89,6 +159,27 @@ std::unique_ptr<column> transform(column_view const& input,
   return output;
 }
 
+std::unique_ptr<column> generalized_masked_op(table_view data_view,
+                                              std::string const& udf,
+                                              data_type output_type,
+                                              rmm::mr::device_memory_resource* mr)
+{
+  rmm::cuda_stream_view stream   = rmm::cuda_stream_default;
+  std::unique_ptr<column> output = make_fixed_width_column(output_type, data_view.num_rows());
+  std::unique_ptr<column> output_mask =
+    make_fixed_width_column(cudf::data_type{cudf::type_id::BOOL8}, data_view.num_rows());
+
+  mutable_column_view output_view      = *output;
+  mutable_column_view output_mask_view = *output_mask;
```

Comment:
This conversion should be implicit when calling `generalized_operation`.

Reply:
Should be fixed